tests/phpunit/CiviTest/bootstrap.php

```diff
@@ -74,7 +74,8 @@ function _phpunit_mockoloader($prefix, $base_dir, $class) {
  *   If the command terminates abnormally.
  */
 function cv($cmd, $decode = 'json') {
-  $cmd = 'cv ' . $cmd;
+  // If xdebug is active when launching phpunit, we usually want to focus on phpunit.
+  $cmd = 'env XDEBUG_MODE=off XDEBUG_PORT= cv ' . $cmd;
```

Comment:
```suggestion
  if (strtoupper(substr(PHP_OS, 0, 3)) !== 'WIN') {
    $cmd = 'env XDEBUG_MODE=off XDEBUG_PORT= cv ' . $cmd;
  }
  else {
    $cmd = 'cv ' . $cmd;
  }
```

But also, won't this prevent step-debugging a unit-test?

Reply:
Good catch re: PHP_OS.